augur/types.py

```diff
@@ -11,3 +11,24 @@ class DataErrorMethod(enum.Enum):
     ERROR_ALL       = 'error_all'
     WARN            = 'warn'
     SILENT          = 'silent'
+
+
+@enum.unique
+class ValidationMode(enum.Enum):
+    """
+    Enum representation of string values that represent how validation should
+    be handled.
+    """
+    ERROR = 'error'
+    WARN  = 'warn'
+    SKIP  = 'skip'
+
+    def __str__(self) -> str:
+        """
+        Stringify to the enum member's :py:attr:`.value` instead of the default.
+
+        This let us use the enum's constructor and members with argparse's
+        ``type`` and ``choices`` parameters, respectively, without exposing the
+        enum class name to users.
+        """
+        return self.value
```

Comment:
Sure! I figured I'd leave that + using it for the curate-used enum for another PR, though.

Also, if we had a minimum dep of Python 3.11, we could just use [StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum).

Reply:
Yup, totally. I'll probably add this change to my current work with `curate format-dates` because I was just thinking to myself that the `DataErrorMethod` enum was annoying to use for argparse.